modules/fb_instant_articles_display/src/DrupalInstantArticleDisplay.php

```diff
@@ -2,9 +2,11 @@
 
 /**
  * @file
- * Defines the InstantArticleNodeWrapper class.
+ * Contains \Drupal\fb_instant_articles_display\DrupalInstantArticleDisplay.
  */
 
+namespace Drupal\fb_instant_articles_display;
```

Comment:
@m4olivei  Nice catch. Fixed in 1e288a4 (I did them all, not just within the namespaced classes).